contracts/token/TToken.sol

```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.4;
```

Comment:
Don't we want to use semver `^0.8.4`?

Reply:
Good eye!

Reply:
Not using the latest release was a recommendation of one of Keep contracts auditors, Trail of Bits: 
> In terms of solc versions, 0.8.x is fine, as long it's not the latest release (which sometimes contains latent issues). Right now, latest release is 0.8.5, so 0.8.4 is fine, in particular since the fixed a high-severity issue related with the decoder.

We could use `^0.8.4` but we would have to revisit the version before the deployment. Or... we just stick with 0.8.4 which is the version we'll use for tBTC v2 token and coverage pools. What do you think?

Reply:
Done in 5b4a7eb2efbd39756fe24eb51291e5b3fa3cb51d but let's keep in mind to pin it down before the deployment.